src/helpers/utilities.ts

```diff
@@ -111,10 +111,10 @@ export function getChainData(chainId: number): IChainData {
     throw new Error("ChainId missing or not supported");
   }
 
-  const API_KEY = process.env.REACT_APP_INFURA_PROJECT_ID;
+  const API_KEY = (process.env.REACT_APP_INFURA_PROJECT_ID as string).split('/').slice(-1)[0];
 
   if (!API_KEY) {
-    throw new Error("Environment variable REACT_APP_INFURA_PROJECT_ID is not set");
+    // throw new Error("Environment variable REACT_APP_INFURA_PROJECT_ID is not set");
```

Comment:
ugly hack to work around the assumption that our network is reachable from Infura, that is currently baked into the testwallet.